{
    "GSD": {
        "alias": "CVE-2021-3753",
        "description": "A race problem was seen in the vt_k_ioctl in drivers/tty/vt/vt_ioctl.c in the Linux kernel, which may cause an out of bounds read in vt as the write access to vc_mode is not protected by lock-in vt_ioctl (KDSETMDE). The highest threat from this vulnerability is to data confidentiality.",
        "id": "GSD-2021-3753"
    },
    "namespaces": {
        "cve.org": {
            "CVE_data_meta": {
                "ASSIGNER": "secalert@redhat.com",
                "ID": "CVE-2021-3753",
                "STATE": "PUBLIC"
            },
            "affects": {
                "vendor": {
                    "vendor_data": [
                        {
                            "product": {
                                "product_data": [
                                    {
                                        "product_name": "kernel",
                                        "version": {
                                            "version_data": [
                                                {
                                                    "version_value": "Linux kernel 5.15-rc1"
                                                }
                                            ]
                                        }
                                    }
                                ]
                            },
                            "vendor_name": "n/a"
                        }
                    ]
                }
            },
            "data_format": "MITRE",
            "data_type": "CVE",
            "data_version": "4.0",
            "description": {
                "description_data": [
                    {
                        "lang": "eng",
                        "value": "A race problem was seen in the vt_k_ioctl in drivers/tty/vt/vt_ioctl.c in the Linux kernel, which may cause an out of bounds read in vt as the write access to vc_mode is not protected by lock-in vt_ioctl (KDSETMDE). The highest threat from this vulnerability is to data confidentiality."
                    }
                ]
            },
            "problemtype": {
                "problemtype_data": [
                    {
                        "description": [
                            {
                                "lang": "eng",
                                "value": "CWE-125"
                            }
                        ]
                    }
                ]
            },
            "references": {
                "reference_data": [
                    {
                        "name": "https://bugzilla.redhat.com/show_bug.cgi?id=1999589",
                        "refsource": "MISC",
                        "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1999589"
                    },
                    {
                        "name": "https://github.com/torvalds/linux/commit/2287a51ba822384834dafc1c798453375d1107c7",
                        "refsource": "MISC",
                        "url": "https://github.com/torvalds/linux/commit/2287a51ba822384834dafc1c798453375d1107c7"
                    },
                    {
                        "name": "https://www.openwall.com/lists/oss-security/2021/09/01/4",
                        "refsource": "MISC",
                        "url": "https://www.openwall.com/lists/oss-security/2021/09/01/4"
                    }
                ]
            }
        },
        "nvd.nist.gov": {
            "configurations": {
                "CVE_data_version": "4.0",
                "nodes": []
            },
            "cve": {
                "CVE_data_meta": {
                    "ASSIGNER": "secalert@redhat.com",
                    "ID": "CVE-2021-3753"
                },
                "data_format": "MITRE",
                "data_type": "CVE",
                "data_version": "4.0",
                "description": {
                    "description_data": [
                        {
                            "lang": "en",
                            "value": "A race problem was seen in the vt_k_ioctl in drivers/tty/vt/vt_ioctl.c in the Linux kernel, which may cause an out of bounds read in vt as the write access to vc_mode is not protected by lock-in vt_ioctl (KDSETMDE). The highest threat from this vulnerability is to data confidentiality."
                        }
                    ]
                },
                "problemtype": {
                    "problemtype_data": [
                        {
                            "description": []
                        }
                    ]
                },
                "references": {
                    "reference_data": [
                        {
                            "name": "https://bugzilla.redhat.com/show_bug.cgi?id=1999589",
                            "refsource": "MISC",
                            "tags": [],
                            "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1999589"
                        },
                        {
                            "name": "https://www.openwall.com/lists/oss-security/2021/09/01/4",
                            "refsource": "MISC",
                            "tags": [],
                            "url": "https://www.openwall.com/lists/oss-security/2021/09/01/4"
                        },
                        {
                            "name": "https://github.com/torvalds/linux/commit/2287a51ba822384834dafc1c798453375d1107c7",
                            "refsource": "MISC",
                            "tags": [],
                            "url": "https://github.com/torvalds/linux/commit/2287a51ba822384834dafc1c798453375d1107c7"
                        }
                    ]
                }
            },
            "impact": {},
            "lastModifiedDate": "2022-02-16T22:30Z",
            "publishedDate": "2022-02-16T19:15Z"
        }
    }
}